ui/main.qml

```diff
@@ -124,8 +124,8 @@ StatusWindow {
 
     Connections {
         id: windowsOsNotificationsConnection
-        enabled: false
-        target: Global.mainModuleInst
+        enabled: Qt.platform.os === Constants.windows
+        target: Global.mainModuleInst || null
```

Comment:
Will this update to work when we login? Since `mainModuleInst` doesn't exist before the login, we want to make sure the connection gets hooked once we do login.

Reply:
Yeah... I have my doubts too; what about instead initializing `mainModuleInst` to `null`?

```qml
property var mainModuleInst: null
```

Reply:
What we have already in place in Global.qml:
```qml
property var globalUtilsInst: typeof globalUtils !== "undefined" ? globalUtils : null
```

Reply:
The reason is called "mock" and StoryBook :) cc @micieslak

Reply:
oh okay, never did it :D